setup.py

```diff
@@ -42,7 +42,7 @@ def filter_requirement_statements(req):
 STATIC_REQUIREMENTS = []
 
 # Decide if the invoked command is a request to do building
-DIST_BUILDING_COMMAND = any([x in sys.argv for x in ("bdist", "sdist", "bdist_wheel")])
+DIST_BUILDING_COMMAND = any([x in sys.argv for x in ("bdist", "sdist", "bdist_wheel", "bdist_deb", "sdist_dsc")])
```

Comment:
It makes setup.py aware of whether it's building or installing... when it's building, it can import "pip" but when it's installing, pip might not be available... only setuptools.